website/content/en/docs/concepts/nodeclasses.md

```diff
@@ -731,6 +731,27 @@ alias: bottlerocket@v1.20.4
 ```
 The Windows family does not support pinning, so only `latest` is supported.
 
+The following commands can be used to determine the versions availble for an alias in your region:
```

Comment:
Should we ask folks to define K8S_VERSION and use the latest like we do in the geting started guide by default so that the command is straight copy-able

Reply:
Ah good point, I'll add that 👍